tests/CMakeLists.txt

```diff
@@ -104,11 +104,15 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 #----------------------------------
 
 add_executable(sdbus-c++-unit-tests ${UNITTESTS_SRCS})
-target_compile_definitions(sdbus-c++-unit-tests PRIVATE LIBSYSTEMD_VERSION=${LIBSYSTEMD_VERSION})
+target_compile_definitions(sdbus-c++-unit-tests PRIVATE
+    LIBSYSTEMD_VERSION=${LIBSYSTEMD_VERSION}
+    SDBUS_HEADER=<${LIBSYSTEMD}/sd-bus.h>)
```

Comment:
Do we need this? I don't see sd-bus headers to be used anywhere in the unit and integration tests...

Reply:
It's used indirectly:
```c++
In file included from tests/unittests/Connection_test.cpp:28:
In file included from src/Connection.h:32:
src/IConnection.h:31:10: error: expected "FILENAME" or <FILENAME>
#include SDBUS_HEADER
         ^
src/IConnection.h:61:59: error: unknown type name 'sd_bus_vtable'
                                                  , const sd_bus_vtable* vtable
                                                          ^
[...]
```

Reply:
OK, we only need this for unit tests.